libbeat/tests/compose/wrapper.go

```diff
@@ -156,7 +159,7 @@ func (d *wrapperDriver) Close() error {
 }
 
 func (d *wrapperDriver) cmd(ctx context.Context, command string, arg ...string) *exec.Cmd {
-	var args []string
+	args := make([]string, 0, 4+len(d.Files)+len(arg)) // preallocate as much as possible
```

Comment:
How will this pre allocation help?

Reply:
Reason for this change:

- Pointed out by CI's golangci-lint (linter used: https://github.com/alexkohler/prealloc)
- Preallocation helps in lesser memory allocations. Check this out: https://github.com/alexkohler/prealloc#purpose (non prealloc vs prealloc version benchmarked).


Explaination:
```
args := make([]string, 0, 4+len(d.Files)+len(arg)) // preallocate as much as possible
args = append(args, "--no-ansi", "--project-name", d.Name) // 3 elements added
for _, f := range d.Files {                                                               // len(d.Files) elements added
	args = append(args, "--file", f)
}
args = append(args, command)                                                  // 1 element added
args = append(args, arg...)                                                          // len(arg) element added

// 3 + len(d.Files) + 1 + len(arg) = 4 + len(d.Files) + len(arg)
```

It is considered good practice to preallocate (by making the slice of the required capacity) in order to reduce memory allocation.